python/sglang/srt/mem_cache/memory_pool_host.py

```diff
@@ -172,6 +172,7 @@ def alloc(self, need_size: int) -> Optional[torch.Tensor]:
 
     @synchronized
     def free(self, indices: torch.Tensor) -> int:
+        indices = indices.unique()
```

Comment:
Why is unique called here? Are there scenarios where the indices might not be unique?